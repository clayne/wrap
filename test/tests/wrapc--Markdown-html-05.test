wrapc | /dev/null | -u | md-html-05.c | 0
