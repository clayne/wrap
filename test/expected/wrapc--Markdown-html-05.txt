  // This is the first sentence.  This is the second sentence.  This is the
  // third sentence.  <em>Span-level HTML.</em> This is the fourth sentence.
  // This is the fifth sentence.  This is the sixth sentence.
